labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise02.scala

```diff
@@ -11,7 +11,11 @@ object ListManipulationExercise02 {
    * As usual, various ways exist: pattern matching, folding, ...
    */
   def maxElementInList(l: List[Int]): Int = {
-    error("fix me")
+    l match{
+      case List(x:Int) => x
+      case x :: y :: xs => maxElementInList((if (x>y) x else y)::xs)
+      case nil => throw new IllegalArgumentException("empty list")
```

Comment:
IIRC if you don't include it, scala will throw a failed pattern match error or something, but this is nice also. 

Also, is `nil` the same as `Nil`?